docs/kubernetes.rst

```diff
@@ -126,6 +126,19 @@ Kubernetes Operator
          }
     ]
 
+    life_cycle = {
+      'postStart': {
```

Comment:
yeah we could potentially have `post_start_command` or `post_start_http` as options rather than having users give a json object. cc: @ashb for thoughts

Reply:
Dunno - it's probably easiest if we mirror the Kube YAML structure isn't it? One less thing to learn, and we can point at the Kube docs rather than having to write our own?

Reply:
@ashb good point. I'm also wondering if we should start having users just build objects using the kubernetes python SDK. It would be far more future proof